membership-attribute-service/app/configuration/Config.scala

```diff
@@ -36,6 +37,10 @@ object Config {
     val region = Regions.EU_WEST_1
   }
 
+  object authentication {
+    val key = config.getString("authentication.key")
```

Comment:
It will be easier to rotate keys if you allow multiple keys to be valid at a time.  A simple way to do this would be `config.getString("authentication.keys").split(',')`